frontend/views/pages/Join.vue

```diff
@@ -160,6 +162,13 @@ export default ({
     goHome () {
       this.$router.push({ path: '/' })
     },
+    acceptAfterLogin () {
+      if (syncFinished) {
+        this.accept()
+      } else {
+        sbp('okTurtles.events/once', LOGIN, () => this.accept())
+      }
+    },
```

Comment:
I don't think this change is needed.

Reply:
Yeah, I can see that in the latest `e2e-protocol` branch, the `login` function is updated inside `LoginForm.vue`.
Since the `login` function is updated to handle login process properly, I don't think this `acceptAfterLogin` function is needed anymore. I will revert this.

> I wasn't able to reproduce the problems in https://github.com/okTurtles/group-income/issues/1715 e2e-protocol branch... maybe latest changes fixed it?

Yes, I can say it is fixed in the latest `e2e-protocol` branch.